ister_gui.py

```diff
@@ -1007,65 +998,145 @@ def _umount_host_disk(self, root_part, boot_part):
+            mask_res = 'none found'
+            ip_res = 'none found'
+
+        dns_res = self.find_dns()
+        if not dns_res:
+            dns_res = 'none found'
+
+        fmt = '{0:>20}'
+        interface = urwid.Text(fmt.format('Interface: ') + interface_res)
+        static_ip = urwid.Text(fmt.format('IP address: ') + ip_res)
+        mask = urwid.Text(fmt.format('Subnet mask: ') + mask_res)
+        dns = urwid.Text(fmt.format('DNS: ') + dns_res)
+        # pylint: disable=E1103
+        try:
+            gateway = netifaces.gateways()['default'][netifaces.AF_INET][0]
+        except Exception:
+            gateway = 'none found'
+
+        gateway = urwid.Text(fmt.format('Gateway: ') + gateway)
+        self.title = urwid.Columns([self.static_header,
+                                    self.detected_header])
+        self.iface = urwid.Columns([self.interface_e, interface])
+        self.static_ip = urwid.Columns([self.static_ip_e, static_ip])
+        self.mask = urwid.Columns([self.mask_e, mask])
+        self.gateway = urwid.Columns([self.gateway_e, gateway])
+        self.dns = urwid.Columns([self.dns_e, dns])
+
+        self.widgets = [self.title,
+                        urwid.Divider()]
+        if self.allow_iface:
+            self.widgets.append(self.iface)
+
+        self.widgets.extend([self.static_ip,
+                             self.mask,
+                             self.gateway,
+                             self.dns,
+                             urwid.Divider(),
+                             self.static_button,
+                             urwid.Divider()])
+
     def set_static_edits(self):
         """Set the edit captions for the static configuration section"""
         fmt = '{0:>20}'
         self.interface_e = urwid.Edit(fmt.format('Interface: '))
         self.static_ip_e = urwid.Edit(fmt.format('IP address: '))
         self.mask_e = urwid.Edit(fmt.format('Subnet mask: '))
         self.gateway_e = urwid.Edit(fmt.format('Gateway: '))
-        self.dns_e = urwid.Edit(fmt.format('DNS (optional): '))
+        self.dns_e = urwid.Edit(fmt.format('DNS: '))
         urwid.connect_signal(self.interface_e, 'change', self._activate_button)
         urwid.connect_signal(self.static_ip_e, 'change', self._activate_button)
         urwid.connect_signal(self.mask_e, 'change', self._activate_button)
         urwid.connect_signal(self.gateway_e, 'change', self._activate_button)
         urwid.connect_signal(self.dns_e, 'change', self._activate_button)
 
+    def find_interface_ip(self):
+        """Find active interface and ip address"""
+        # pylint: disable=E1103
+        addrs = {}
+        mask = ''
+        af_inet = netifaces.AF_INET
+        for if_name in netifaces.interfaces():
+            ifaddrs = netifaces.ifaddresses(if_name)
+            if af_inet in ifaddrs:
+                ip_addrs = ifaddrs[af_inet][0]
+                if ip_addrs['addr'] and '127.0.' not in ip_addrs['addr']:
```

Comment:
That was just me trying to be more specific in case 127 occurred somewhere else in the ip, but this would be better as

```python
if ip_addrs['addr'] and not ip_addrs['addr'].startswith('127'):
```